code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -683,7 +683,7 @@
 /obj/effect/proc_holder/changeling/gloves/gauntlets/sting_action(mob/living/user)
 	if(HAS_TRAIT(user, TRAIT_NOPUGILIST))
 		to_chat(user, "<span class='warning'>We would gain nothing by forming our fists into brute-force weapons when we are trained in precision martial arts!</span>")
-		return
```

Comment:
Leave this return in though so the if still has an effect.

Reply:
Done